src/pages/Home/Home.jsx

```diff
@@ -16,23 +18,24 @@ export const Home = () => {
 					</div>
 				</section>
 				<section className="note_lisiting_section">
-					<h3>Pinned Notes</h3>
-					<NoteListWrapper>
-						{noteList?.length > 0 ? (
-							<NoteListing
-								list={noteList.filter((note) => note.isPinned)}
-								isPinSection={true}
-							/>
-						) : null}
-					</NoteListWrapper>
+					{pinnedList?.length > 0 ? (
+						<>
+							<h3>Pinned Notes</h3>
+							<NoteListWrapper>
+								<NoteListing list={pinnedList} isPinSection={true} />
+							</NoteListWrapper>
+						</>
+					) : null}
```

Comment:
why are you using null here you could have used short-circuiting for this

Reply:
We use short-circuiting in javaScript as it evaluates number or any data type to a boolean, but in case of React, specifically JSX, numbers or any html valid text would directly be rendered to view and won't evaluate to boolean, which would break our logic. In this case it was not necessary to use ternary operator and short-circuiting would had worked fine. 
But in-order to avoid such errors i am consistently using ternary rather than short-circuiting.

for detailed explanation, you could check this  article :- https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx